### ISSUE TITLE
[Question]: Can I finetune or training from scratch the recognition model?

### ISSUE THREAD

**Issue by HuuHuy227** [infiniflow/ragflow#944]
### Describe your problem

I want to finetune or training from scratch the recognition model from my dataset on my own language. I wonder whether i can do that and how to do that.

**Comment by KevinHuSh** [comment]
Do you mean layout recognition?
We will open source the trainning method in the near future.

**Comment by HuuHuy227** [comment]
> Do you mean layout recognition? We will open source the trainning method in the near future.

No, I mean text recognition. Layout, detect and table worked well in my case.
